templates/stub.logsearch-for-cloudfoundry.yml

```diff
@@ -58,3 +69,11 @@ properties:
   syslog:
     host: (( grab jobs.ingestor.networks.[0].static_ips.[0] ))
     port: 5514
+  smoke_tests:
+    api: (( grab properties.cloudfoundry.api_endpoint ))
+    apps_domain: (( grab properties.cf-kibana.apps_domain ))
+    admin_user: admin
```

Comment:
Actually `firehose_password` and `firehose_user` maybe not the right choice. Smoke-tests require exactly admin permissions, when firehose-to-syslog need only `doppler.firehose` scope. I will change to `firehose_user` because usually this is admin user, but we should notice that this two users require different scope values.